### PR TITLE
Create Product Card molecule

### PR DESCRIPTION
**Pull request by RDO34**
<div align="center"></div>

### CODE REVIEW

**Comment by spn1** [comment]
<div align="right"></div>
<div align="right">😎</div>

change blur to overlay thingy

Make sure image doesnt overlap with the text

File: src/components/molecules/product-card/components/Card.jsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    background-size: cover;
+    display: block;
+    flex-flow: column;
+    overflow: hidden;
+    position: absolute;
+    transition: 0.3s all;
+    top: -20px;
+    left: 0px;
+    width: 275px;
+    height: 400px;
```
Comment (spn1) [inline src/components/molecules/product-card/components/Card.jsx]: could we get some constants for these common values

File: src/components/molecules/product-card/components/UnavailabilityMessage.jsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+export const UnavailabilityMessage = () => {
```
Comment (spn1) [inline src/components/molecules/product-card/components/UnavailabilityMessage.jsx]: is a bit misaligned 👀 
image

File: src/components/molecules/product-card/components/UnavailabilityMessage.jsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    transition: 0.3s all;
+    width: 50%;
+
+    @media screen and (min-width: 1024px) {
+      opacity: 0;
+    }
+  `;
+
+  return (
+    <Message className="c-product-card__unavailability-message">
```
Comment (spn1) [inline src/components/molecules/product-card/components/UnavailabilityMessage.jsx]: why do we use emotion, only to give it classes anyway 🤔 (i dont know emotion that well 🤷‍♂)

**Review by spn1 — COMMENTED** [review]
LGTM but saw a few things 🌮

File: src/components/molecules/product-card/components/UnavailabilityMessage.jsx (hunk 2 above)
Comment (RDO34) [reply src/components/molecules/product-card/components/UnavailabilityMessage.jsx]: 🤔 it doesn't look like that for me? Guessing it's to do with the font, are you using windows?

File: src/components/molecules/product-card/components/Card.jsx (hunk 1 above)
Comment (RDO34) [reply src/components/molecules/product-card/components/Card.jsx]: 👍

File: src/components/molecules/product-card/components/UnavailabilityMessage.jsx (hunk 3 above)
Comment (RDO34) [reply src/components/molecules/product-card/components/UnavailabilityMessage.jsx]: This seemed like the best way to create the hover effects controlled by the container element, open to other suggestions tho

File: src/components/molecules/product-card/components/Card.jsx (hunk 1 above)
Comment (RDO34) [reply src/components/molecules/product-card/components/Card.jsx]: Stuck the internal values (ie card height and width) into a constants file. Left global style constants, such as colours or fonts, for another PR.

File: src/components/molecules/product-card/components/UnavailabilityMessage.jsx (hunk 2 above)
Comment (spn1) [reply src/components/molecules/product-card/components/UnavailabilityMessage.jsx]: damn windows
https://i.giphy.com/media/1Syi8XSAkiFupEty1u/giphy.webp

File: src/components/molecules/product-card/ProductCard.stories.js [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+          displayName: text('Display Name', 'Jacket Potatoe'),
+          currentPrice: number('Current Price', 50),
+          imgSrc: text('Image Source', exampleImgSrc),
+          inStock: boolean('In Stock?', true)
+        }}
+        href={text('href', '/')}
+        onClick={select('onClick', [() => {}, null], null)}
+      />
+    ),
+    { notes: { markdown: readMe } }
```
Comment (snewton27) [inline src/components/molecules/product-card/ProductCard.stories.js]: image

**Review by snewton27 — APPROVED** [review]
🌮

File: src/components/molecules/product-card/ProductCard.stories.js (hunk 4 above)
Comment (snewton27) [reply src/components/molecules/product-card/ProductCard.stories.js]: rEaDMe

**Pull request merged**